src/controllers/fxaRpEvents.js

```diff
@@ -0,0 +1,167 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as jwt from 'jsonwebtoken'
+import jwkToPem from 'jwk-to-pem'
+import { captureException } from '@sentry/node'
+import { UnauthorizedError } from '../utils/error.js'
+import { deleteSubscriberByFxAUID, getSubscriberByFxaUid, updateFxAProfileData } from '../db/tables/subscribers.js'
+import mozlog from '../utils/log.js'
+import appConstants from '../appConstants.js'
+const log = mozlog('controllers.fxa-rp-events')
+
+const FXA_PROFILE_CHANGE_EVENT = 'https://schemas.accounts.firefox.com/event/profile-change'
+const FXA_SUBSCRIPTION_CHANGE_EVENT = 'https://schemas.accounts.firefox.com/event/subscription-state-change'
+const FXA_DELETE_USER_EVENT = 'https://schemas.accounts.firefox.com/event/delete-user'
+
+/**
+ * Fetch FxA JWT Public for verification
+ *
+ * @returns {Array} keys an array of FxA JWT keys
+ */
+const getJwtPubKey = async () => {
+  jwtKeyUri = `${appConstants.OAUTH_ACCOUNT_URI}/jwt`
+  try {
+    const res = await fetch(jwtKeyUri, {
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    })
+    const { keys } = res
+    log.info('getJwtPubKey', `fetched jwt public keys from: ${jwtKeyUri} - ${keys}`)
+    return keys
+  } catch (e) {
+    captureException('Could not get JWT public key', jwtKeyUri)
```

Comment:
I think this method is intended for exceptions not strings: https://docs.sentry.io/platforms/javascript/usage/

Reply:
I wonder if it sends an event to Sentry the same way that captureExcpetion would... Another option is to wrap the message in an `new Error`. I'll test it out.